libeos-updater-flatpak-installer/perform-flatpak-actions.c

```diff
@@ -440,6 +456,20 @@ check_if_flatpak_is_installed (FlatpakInstallation        *installation,
   return TRUE;
 }
 
+/**
+ * eufi_check_ref_actions_applied:
+ * @installation: a #FlatpakInstallation
+ * @pending_flatpak_deployments_state_path: unused
```

Comment:
It could be a bug. If that parameter was there for anything, it would be there to limit the checked actions to include the ones up to and including the serials specified in that path. Perpahps better to just remove it though